app/views/users/_followers.html.haml

```diff
@@ -1,6 +1,10 @@
 %article
   - if @user.followers.empty?
     %p No one is following #{@user.username} yet.
-  - @user.followers.each do |follower|
-    %p= avatar(follower.email)
-    = follower.username
\ No newline at end of file
+    
+  -@user.followers.each do |follower|
+    = avatar(follower.email)
+    - if follower.username == current_user.username   
+      = "You"
+    - else
```

Comment:
Hey, why have you used `if follower.username == current_user.username`? A user can never follow himself (if you found a case where he was then we would need to fix it)  
Also, I don't know if you noticed but you are failing some of them haml-lint rules. You might want to fix it.